### ISSUE TITLE
Results are very poor

### ISSUE THREAD

**Issue by engmubarak48** [ee09115/spoofing_detection#16]
Hi thanks for providing your hardwork. 

But results needs to be improved, do you have any suggestions for further performance improvement. 

thanks

**Comment by bcba25** [comment]
@engmubarak48 hi, any progress on this ? did u manage to improve it in any way ?

**Comment by engmubarak48** [comment]
> @engmubarak48 hi, any progress on this ? did u manage to improve it in any way ?

@hxdef2517 sorry for the late reply, no I left this. kind of working on another one.

**Comment by lsvannur** [comment]
very poor results

**Comment by ankitnamdeo34** [comment]
agreed the results are very poor, I also have started working on the same type of problem to identify whether the picture taken is of the live person or it is taken from some other photo. If anyone is interested please do let me know, will really be happy to discuss.

**Comment by engmubarak48** [comment]
@ankitnamdeo34 I am very interested, let me know if you find a data or pretrained model.

**Comment by sybrin-innovations** [comment]
@engmubarak48 @ankitnamdeo34. Hey guys i also start working on a solution to identify fakes. If you would like to discuss further please drop me a mail. Maybe we can help each other

**Comment by Coderx7** [comment]
@engmubarak48 @ankitnamdeo34 , @sybrin-innovations any update guys?

**Comment by Bagdu** [comment]
Hi, i have a same problema to solve,  where can we talk  about it ?

**Comment by yesgvinayak** [comment]
@ankitnamdeo34 @sybrin-innovations @engmubarak48  , guys, I am also working on the same topic. Did you guys have any progress on this one or Can you help me with something that you are using for this one.

**Comment by Madina-S** [comment]
@ankitnamdeo34 @sybrin-innovations @engmubarak48 @yesgvinayak Hi there, I have the same issue, please provide some guideline.
